src/requests/team.ts

```diff
@@ -366,6 +370,30 @@ async function getIntegrations({
   return decryptedIntegrations;
 }
 
+export async function getTeamSettings(client: GraphQLClient, slug: string) {
+  const response: { team_settings: TeamSettings[] } = await client.request(
+    gql`
+      query GetTeamSettings($slug: String!) {
+        team_settings(where: { team: { slug: { _eq: $slug } } }) {
```

Comment:
```suggestion
  const response: { teamSettings: TeamSettings[] } = await client.request(
    gql`
      query GetTeamSettings($slug: String!) {
        teamSettings: team_settings(where: { team: { slug: { _eq: $slug } } }) {
```

Wherever possible we want to keep all out TypeScript using camelCase and do our casting / destructing from snake_case within the GraphQL query.

Reply:
Thanks for picking this up!